documentation/user_docs/README.md

```diff
@@ -0,0 +1,43 @@
+# Offline Documentation
+
+## Building Docs
+
+### Requirements
+The documentation requires the following to be installed:
+* Python (3.5+)
+* Sphinx
+* Sphinx RTD theme
+
+On Linux (Optional):
+* Make
+
+### Building Locally
+First, clone the documentation repository (https://github.com/pace-neutrons/horace-docs)
+
+    git clone https://github.com/pace-neutrons/horace-docs.git
+
+* Windows
+
+  With a command terminal in the root folder of the horace-docs project run:
+
+      make.bat html
+
+* Linux (With Make)
+
+  With `make` installed simply run:
+
+      make html
+
+* Linux (No Make)
+
+  If you do not have `make` installed, run:
+
+      sphinx-build -M html "docs" "build"
+
+
+Built documents will be put in the `/build/html` folder, the main file is the `index.rst` file.
+
+## Online documentation
+Horace documentation is also available from:
+
+https://pace-neutrons.github.io/horace-docs/3.5.0/ 
```

Comment:
It's not clear this is an example and the general form is `/<version>` or `/latest`. We don't want to have to update this document with each new release

Reply:
As we haven't set up the files for /latest /stable, etc. I haven't set that yet. We probably need to decide on a name. I would argue "latest" and "stable" personally for current dev and last release respectively.

Reply:
I'd extend the "online documentation" section here to reference the a least two of those options:
- Latest release: https://pace-neutrons.github.io/horace-docs
- Direct link to a specific version: https://pace-neutrons.github.io/horace-docs/<version> e.g. https://pace-neutrons.github.io/horace-docs/3.5.0 

I'm in two minds whether we should publicise the link to nightly build documentation, There's probably no harm in it.
 
If we can set up a redirect from `https://pace-neutrons.github.io/horace-docs` to `https://pace-neutrons.github.io/horace-docs/latest` that would be nice usability win.